chcemvediet/apps/inforequests/forms/obligee_action.py

```diff
@@ -202,10 +202,10 @@ def post_transition(self):
 
         return res
 
-class DisclosureLevelFork2(ObligeeActionStep):
+class DisclosureLevelFork(ObligeeActionStep):
```

Comment:
@martinmacko47 FYI: Wizard nam umoznuje z akcie `Full-Disclosure` pridat novu akciu `Non-Full Disclosure` a naopak. Je to tak aj v tabulke. Podla grafu  `inforequest-flow.svg` to vsak nie je mozne, nie su navzajom prepojene sipkami.

Reply:
@viliambalaz Povolit "Full-Disclosure" po "Non-Full Disclosure" imho mozne. Institucia moze poslat odpoved vo viacerych mailioch po sebe, takze tie maily budu postupne "Non-Full Disclosure" az posledny mail, ktory zodpovie vsetko chybajuce bude "Full-Disclosure".

Naopak "Non-Full Disclosure" po "Full-Disclosure" je dost matuce. Ale neviem ako to rozumne zakazat, pretoze v property `branch.can_add_disclosure` nemame k dispozicii disclosure level movej akcie, iba starej. A vyrabat oddelene properties `branch.can_add_full_disclosure` a `branch.can_add_non_full_disclosure` by dost komplikovalo kod. Kedze potom by sme museli mat dve verzie aj pre `branch.can_add_disclosure_refusal_advancement_extension`, a metoda `branch.can_add_action` by potrebovala parameter, aka akcia sa ide pridat, nie len typ tej akcie. Resp. by sme museli uplne lowlevelovo rozdelit action type `DISCLOSURE` na dve `FULL_DISCLOSURE` a `NON_FULL_DISCLOSURE`. Co by bolo vela prerabania. Preto radsej pre jednoduchost "Non-Full Disclosure" po "Full-Disclosure" povolme tiez, a nechajme na zodpovednosti usera, ze po "Full-Disclosure" uz nebude tvrdit, ze zrazu je odpoved neuplna.

Cize wizard nech teda povoluje "Full-Disclosure" po "Non-Full Disclosure" a naopak, tak ako to mame spravene.

Zislo by sa vsak do "Inforequest Flow" obrazku zakreslit, ze po "Full-Disclosure" moze zasledovat "Non-Full Disclosure" a naopak. Skus to v samostatnom PR nejak do toho obrazku zakreslit tak, aby to nebolo moc chaoticke a matuce.

Reply:
Spravime v samostatnom PR. Presunute do https://github.com/martinmacko47/chcemvediet/issues/172#issuecomment-569065059